<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSSH</class>
 <widget class="QWidget" name="EditSSH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditSSH</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="user_l">
     <property name="text">
      <string>User</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="user"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="password_l">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="password"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="private_key_l">
     <property name="text">
      <string>Private Key</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTextEdit" name="private_key"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="private_key_pass_l">
     <property name="text">
      <string>Private Key Password</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="private_key_pass"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="host_key_l">
     <property name="text">
      <string>Host Key</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="host_key"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="host_key_algs_l">
     <property name="text">
      <string>Host Key Algorithms</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="host_key_algs"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="client_version_l">
     <property name="text">
      <string>Client Version</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="client_version"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="private_key_path_l">
        <property name="text">
         <string>Private Key Path</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="private_key_path"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="choose_pk">
        <property name="text">
         <string>Choose File</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
